pandas/tests/frame/test_indexing.py

```diff
@@ -185,6 +186,36 @@ def test_setitem_list_of_tuples(self):
         expected = Series(tuples, index=self.frame.index, name='tuples')
         assert_series_equal(result, expected)
 
+    def test_setitem_list_all_missing_columns_scalar(self):
+        df = self.frame.copy()
+        df[['E', 'F']] = 1
+
+        df2 = self.frame.copy()
+        df2['E'] = 1
+        df2['F'] = 1
+
+        assert_frame_equal(df, df2)
+
+    def test_setitem_list_some_missing_columns_list(self):
+        df = self.frame.copy()
```

Comment:
use float_frame fixture (if you can)

Reply:
What's float_frame fixture? Mind giving me some pointer?

Reply:
have a look at #25633.

the other tests in this module are being updated to use the float_frame fixture.

Reply:
Updated.